frontend/src/components/Helper/AsyncCalls.ts

```diff
@@ -0,0 +1,26 @@
+import { Item } from "../../constants/Interfaces";
+import { GlobalStates } from "../../constants/States";
+import temporaryDB from '../../../temporaryDB.json'; // FIXME: will be removed later
+import { ActionTypes } from "../../constants/ActionTypes";
+
+// All asyncrhonous calls to backend
+// TODO: implement backend calls once backend is completed
+
+// fill in the store information for selected items
```

Comment:
If you change this to the 
```
/**
 * explanation
 */
```
style of comment, VSCode will automatically show the docstring when you hover over the function name, which is handy if you want to quickly check what something does